frontend/components/shelter/home/PetsTable/components/Name.tsx

```diff
@@ -0,0 +1,22 @@
+type TableNameProps = {
+	name: string;
+	image?: string; // TODO: confirm type
+};
+
+const TableName = ({ name, image }: TableNameProps) => {
+	return (
+		<div>
+			{image && (
+				// eslint-disable-next-line @next/next/no-img-element
```

Comment:
can be removed as `"@next/next/no-img-element": "off",` is now in `.eslintrc.json`